sys/include/net/gcoap.h

```diff
@@ -384,6 +401,9 @@
 #include "event/timeout.h"
 #include "net/ipv6/addr.h"
 #include "net/sock/udp.h"
+#if IS_ACTIVE(CONFIG_GCOAP_ENABLE_DTLS)
```

Comment:
Why not 
```suggestion
#if IS_ACTIVE(MODULE_GCOAP_DTLS)
```

Reply:
`s/IS_ACTIVE/IS_USED/` ;-)

Reply:
Given that we have

```C
#define IS_USED(module) IS_ACTIVE(module)
```

we should just get rid of either define - but that's unrelated to this PR

Reply:
Is there any preference regarding the two macros? For modules `IS_USED` sounds a bit more appropriate